DrinkManagerWeb/Startup.cs

```diff
@@ -34,6 +36,18 @@ public void ConfigureServices(IServiceCollection services)
                 })
                 .AddEntityFrameworkStores<DrinkAppContext>();
 
+            services.AddLocalization();
+            services.Configure<RequestLocalizationOptions>(options =>
+            {
+                options.SetDefaultCulture("en-GB");
+                options.AddSupportedCultures("en-GB","pl-PL");
+                options.FallBackToParentUICultures = true;
+
```

Comment:
Here i had to add `options.AddSupportedUICultures("en-GB", "pl-PL");` in order to make it work on my machine.

Reply:
Added